python-engine/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "yggdrasil-engine"
-version = "0.1.6"
+version = "1.0.0"
```

Comment:
why not 1.1.0 as ruby?

Reply:
I see what you mean now that I look this over again though. It's suspect that 2 other engines are being bumped to 1.1.0 here. Promise it's a weird coincidence :laughing:

Reply:
ah right. I got confused by weirdly similar numbers :D